sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -226,12 +226,15 @@ def _convert_span_to_envelope(span: ReadableSpan) -> TelemetryItem:
                 else:
                     data.source = span.attributes[SpanAttributes.MESSAGING_DESTINATION]
         # Apply truncation
-        if data.url:
-            data.url = data.url[:2048]  # Breeze max length
-        if data.response_code:
-            data.response_code = data.response_code[:1024]  # Breeze max length
```

Comment:
Was there a particular reason for this move?

Reply:
Just a reordering based off the order it appears in the schema.